yolo_to_yolo/data_types.py

```diff
@@ -25,11 +26,24 @@ class TargetAnnotation(NamedTuple):
     location: YoloOutline | YoloBbox
     classname: str
 
+class YoloClassType(Enum):
+    SHAPE = 1
+    CHARACTER = 2
+    UNKNOWN = 3
 
 class YoloLabel(NamedTuple):
     location: YoloOutline | YoloBbox
     classname: str
 
+    @property
+    def class_type(self) -> YoloClassType:
```

Comment:
This is the right idea, but it doesn't take care of the color labels, which reminds me that your IOU calculations need to take into consideration color as well, per the original issue:
| The output labels should be the classes that are in the target, namely shape, character, shape color, character color.

I need to ask @EricPedley where we can get data with color labels to see how it's formatted. We'll eventually want to write a data reader that takes raw Godot data (which has color labels) and read it straight into our pipeline, but we're occupied with other things right now.

For now, assume that they're going to be in parallel, as in, there's gonna be two more classes per target, and you have to associate them with the shape or character first, then you can associate the shapes with the characters.